apps/demo/tests/constants.ts

```diff
@@ -1 +1 @@
-export const BASE_URL = process.env['BASE_URL']!
+export const BASE_URL = process.env['BASE_URL'] || 'http://localhost:3000/'
```

Comment:
again should we keep the same port as lab? how would playwright manage that?

Reply:
PW will handle separately 🙏